maxim/logger/openai/completions.py

```
@@ -47,15 +93,20 @@ def create(self, *args, **kwargs):
-                generation.result(OpenAIUtils.parse_completion(response))
-            if is_local_trace and trace is not None:
-                trace.set_output(response.choices[0].message.content or "")
-                trace.end()
-        except Exception as e:
-            scribe().warning(
-                f"[MaximSDK][MaximOpenAIChatCompletions] Error in logging generation: {str(e)}"
-            )
+        if is_streaming:
+            # For streaming responses, return a wrapped stream that handles logging
+            return StreamWrapper(response, generation, trace, is_local_trace)
+        else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)